### ISSUE TITLE
`django-serializable-model` name can't be imported due to hyphens

### ISSUE THREAD

**Issue by dev0088** [agilgur5/django-serializable-model#2]
I added this code In settings.py file after run `pip install django-serializable-model`:

#### settings.py
```python
    INSTALLED_APPS = (
        # ...
        'django-serializable-model',
    )
```

But I had this error:
```
ModuleNotFoundError: No module named 'django-serializable-model'
```

How can I use this package in my Django app? 
I think your package name for importing must not include `-` character.

**Comment by agilgur5** [comment]
Hi @ninjadev999, thanks for reporting this issue :)

There's two different problems here, so let me address them separately:

1. I'm not sure why you're using [`INSTALLED_APPS`](https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-INSTALLED_APPS) as that isn't how this package is used. Not sure if that's just what you were trying after `from django-serializable-model import SerializableModel` failed or if that's how you tried to initially use it.
The installation and usage directions don't mention `INSTALLED_APPS` at all as this isn't a [Django "application"](https://docs.djangoproject.com/en/1.11/ref/applications/#projects-and-applications), it's just an ordinary Python module and is just `import`ed into your code like most Python modules.

2. The second part is totally my mistake. This is actually very confusing in Python-land, because [one does _not_ import a package by its PyPI name](https://stackoverflow.com/a/54599368/3431180), dashes/hyphens are [not encouraged](https://www.python.org/dev/peps/pep-0008/#package-and-module-names), but also [not illegal](https://stackoverflow.com/a/37831973/3431180), despite causing syntax errors in normal usage, and even make up [greater than 25%](https://gist.github.com/gene1wood/9472a9d0dffce1a56d6e796afc6539b8) of PyPI packages. And then we have the conundrum that [underscores, hyphens, and periods are actually treated equivalently by `setuptools`](https://packaging.python.org/guides/distributing-packages-using-setuptools/#name). Suffice to say, Python has not made this easy or intuitive to avoid, and perhaps made it easy to run into instead 😕 

But addressing the question/problem directly, the actual module name of this package is currently [`serializable`](https://github.com/agilgur5/django-serializable-model/blob/dc81fd6cd47ae37db86f2db7601a836598d83386/setup.py#L50), so that means it's imported as:
```python
from serializable import SerializableModel
```
and not as `django-serializable-model` which would cause a syntax error.
I think I noticed this at some point when publishing, but then forgot about it and then repeatedly looked at the README like "something feels wrong here..." 😅 . This was my first Python package, so my apologies for falling into Python's trap of extremely confusing package naming/importing.

You can use the workaround above for now to actually use the module. I'll create a wrapper or mirror module named `django_serializable_model.py`, add that to the `py_modules` config so it can be imported as `django_serializable_model` but without breaking/removing the current `serializable` until a major upgrade, and change the README to use underscores instead of hyphens in a patch release.

**Comment by agilgur5** [comment]
Think I can confirm that changing the `setup.py` `name` field to underscores will not change the PyPI name -- did this in https://test.pypi.org/project/django-serializable-model/0.0.4.dev0/ and it still has the hyphens and will still redirect underscores in the URL name to hyphens.

**Comment by agilgur5** [comment]
@ninjadev999 the module naming issue has been resolved in [v0.0.4](https://github.com/agilgur5/django-serializable-model/releases/tag/v0.0.4). If you re-install with v0.0.4, you should be able to import the module as `django_serializable_model`:
```python
from django_serializable_model import SerializableModel
```

Also sorry if you saw a stray comment here, that was meant for the PR, not this issue, it was moved there.
